src/rgw/setup/rgw.py

```diff
@@ -827,6 +827,22 @@ def _logrotate_generic(conf: MappedConf):
             Log.info(f'{const.LOGROTATE_TMPL} file copied to {const.LOGROTATE_CONF}')
         except Exception as e:
             Log.error(f"Failed to configure logrotate for {const.COMPONENT_NAME}. ERROR:{e}")
+
+        # Copy log rotate script for rgw core files into system's logrotate directory.
+        old_core_logrotate_file = os.path.join(const.LOGROTATE_DIR, const.CORE_LOGROTATE_FILE_NAME)
+        core_dir_path = os.path.join(log_file_path, const.RGW_CORE_FILE_DIR_NAME)
+        if os.path.exists(old_core_logrotate_file):
+            os.remove(old_core_logrotate_file)
+        try:
+            with open(const.CORE_LOGROTATE_TMPL, 'r') as f:
+                content = f.read()
+            content = content.replace('TEMP_CORE_FILE_DIR_PATH', core_dir_path)
+            with open(const.CORE_LOGROTATE_FILE, 'w') as f:
+                f.write(content)
```

Comment:
why this TEMP_CORE_FILE_DIR_PATH maintained in file, which needs to be replaced then

Reply:
this is place holder and kept in this template file. 
During mini-provisioner, the actual path of rgw core file will be replaced here.
This is simillar logic we followed for rgw log rotation code.
this way we are ignoring any hardcoding of log path in our code.

Reply:
ok